### DISCUSSION TITLE
Export and Import Functions

### DISCUSSION THREAD

**Discussion by Web-cloud-warrior** [veler/PaZword#31] [Ideas]
Hi,

I added some functionality to share data between devices, or just to backup the data. This stems from a fundamental distrust of data in webservices.

Once you export the data, to a file you choose, it can be imported on another device.

![expimp](https://user-images.githubusercontent.com/73671065/128276492-1f203238-b244-4d8f-ac2c-a7f364d71ef0.jpg)

**Comment by veler** [comment]
Hi,
This is awesome! Thank you very much for this. Much appreciated :)
Do you have a pull request you could open?
